Neos.Neos/Classes/Domain/Service/UserService.php

```diff
@@ -156,21 +156,28 @@ class UserService
     /**
      * Retrieves a list of all existing users
      *
+     * @param string $sortBy
+     * @param string $sortDirection
      * @return QueryResultInterface The users
      * @api
      */
-    public function getUsers(): QueryResultInterface
+    public function getUsers(string $sortBy, string $sortDirection): QueryResultInterface
     {
+        if ($sortBy === UserRepository::SORT_BY_LASTLOGGEDIN) {
+            return $this->userRepository->findAllOrderedByLastLoggedInDate($sortDirection === UserRepository::SORT_DIRECTION_DESC);
```

Comment:
Instead of passing a boolean and later converting that boolean to DESC / ASC, it would make sense to pass the direction directly